code/modules/mob/inventory.dm

```diff
@@ -169,6 +169,13 @@
 		return hand_index || TRUE
 	return FALSE
 
+/mob/proc/force_hand_swap()

```

Comment:
This skips over the proper procs for equipin and unequiping things from slots, and I don't think this is the right way of handling this.

Reply:
haha get it, handling? ok ill stop.